Pipfile

```diff
@@ -11,11 +11,15 @@ Sphinx = "*"
 [packages]
 awscli = "*"
 pypdf2 = "*"
-ipywidgets = "*"
-farm-haystack = {extras = ["colab", "faiss", "preprocessing"], version = "*"}
-ipynb = "*"
-boto3 = "*"
-python-dotenv = "*"
```

Comment:
I believe some of these dependencies (dotenv, haystak) are still needed by other notebooks in the repo. Even the notebook in this PR uses dotenv, right?